R/github.R

```diff
@@ -53,6 +57,17 @@ github_tag <- function(username, repo, ref = "master") {
   github_GET(file.path("repos", username, repo, "tags", ref))
 }
 
+github_delete_repo <- function(username, repo, pat = github_pat()) {
```

Comment:
I'm moving it into the test file, where it is used and makes it reasonably available for development.